llvm/test/CodeGen/RISCV/rv32zbb.ll

```diff
@@ -1479,3 +1479,215 @@ entry:
   %cmp = icmp ne i32 %popcnt, 1
   ret i1 %cmp
 }
+
+define i8 @sub_if_uge_i8(i8 %x, i8 %y) {
```

Comment:
nit: I understand that in the C code you provided, the comparison is UGE, but could you describe the LLVM IR here instead? which is ULT

Reply:
The ULT just seems to be the canonical form from instcombine